raster/r.viewshed/main.cpp

```diff
@@ -501,7 +501,7 @@ parse_args(int argc, char *argv[], int *vpRow, int *vpCol,
     obsElevOpt->required = NO;
     obsElevOpt->key_desc = "value";
     obsElevOpt->description = _("Viewing elevation above the ground");
-    obsElevOpt->answer = "1.75";
+    obsElevOpt->answer = G_store("1.75");
```

Comment:
I really think that we should use `G_store()` on a case-by-case basis, not globally. If a module never changes `->answer`, simple casting should do just fine. In this case, it's just a warning, not a segfault error from trying to alter static strings. I would prefer `(char *)"1.75"` over `G_store("1.75")`. Maybe, that's just me?

Reply:
This is the only warning for assigning a string to Option->answer, this is because of this being c++ code (e.g. `i.atcorr` already uses G_store). There is no need to change other c code files. I think this is being consistent with existing code.